### PR TITLE
Fix : Projects listing page issue and Download projects issue

### PR DESCRIPTION
**Pull request by vishwanath1004**
Fix : Projects listing page issue and Download projects issue

### CODE REVIEW

File: src/app/manage-learn/project/project-listing/project-listing.component.ts [hunk 1]
```diff
@@ -180,9 +193,12 @@ export class ProjectListingComponent implements OnInit {
     }
 
     fetchProjectList() {
+        this.projects = [];
         if (this.networkFlag) {
+            console.log(this.projects, "this.projects 185", this.networkFlag, this.selectedFilterIndex);
             this.selectedFilterIndex == 1 ? this.getProjectList() : this.getCreatedProjects()
         } else {
+            console.log(this.projects, "this.projects 188", this.networkFlag);
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/project-listing/project-listing.component.ts:201]: rm

File: src/app/manage-learn/project/project-listing/project-listing.component.ts [hunk 2]
```diff
@@ -113,19 +121,22 @@ export class ProjectListingComponent implements OnInit {
...
-        this.networkFlag = this.commonUtilService.networkInfo.isNetworkAvailable;
-        this.projects = [];
-        this._networkSubscription = this.commonUtilService.networkAvailability$.subscribe(async (available: boolean) => {
-            this.clearFields();
-            this.networkFlag = available;
-            this.projects = [];
-            this.fetchProjectList()
-        });
-    }
+    // private initNetworkDetection() {
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/project-listing/project-listing.component.ts:124]: is it not required

File: src/app/manage-learn/project/project-listing/project-listing.component.ts [hunk 3]
```diff
@@ -69,6 +69,14 @@ export class ProjectListingComponent implements OnInit {
             this.filters = [translations['FRMELEMNTS_LBL_CREATED_BY_ME'], translations['FRMELEMNTS_LBL_ASSIGNED_TO_ME']];
             this.selectedFilter = this.filters[0];
         });
+
+        this._networkSubscription = this.commonUtilService.networkAvailability$.subscribe(async (available: boolean) => {
+            this.clearFields();
+            this.networkFlag = available;
+            this.projects = [];
+            console.log(this.projects, "this.projects 123");
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/project-listing/project-listing.component.ts:77]: rm

File: src/app/manage-learn/project/project-listing/project-listing.component.ts [hunk 4]
```diff
@@ -113,19 +121,22 @@ export class ProjectListingComponent implements OnInit {
...
+    //         this.projects = [];
+    //         console.log(this.projects, "this.projects 123");
+    //         this.fetchProjectList();
+    //     });
+    // }
 
     getDownloadedProjectsList() {
+        console.log('in getDownloadedProjectsList');
         this.getDownloadedProjects().then(project => {
+            console.log(project, "getDownloadedProjects 131");
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/project-listing/project-listing.component.ts:139]: rm all logs

File: src/app/manage-learn/project/project-listing/project-listing.component.ts (hunk 2 above)
Comment (vishwanath1004) [reply src/app/manage-learn/project/project-listing/project-listing.component.ts:123]: not required am calling in constructor, bcz of this projects list api calling twice.

**Pull request merged**